src/pkg/services/m365/api/channels_pager.go

```diff
@@ -67,7 +68,12 @@ func (c Channels) NewChannelMessageDeltaPager(
 	}
 
 	options := &teams.ItemChannelsItemMessagesDeltaRequestBuilderGetRequestConfiguration{
-		Headers: newPreferHeaders(preferPageSize(maxNonDeltaPageSize)),
+		QueryParameters: &teams.ItemChannelsItemMessagesDeltaRequestBuilderGetQueryParameters{},
```

Comment:
Is query param required here?

Reply:
Yes, so that we don't hit a nil-pointer on line 76.